userspace/libpman/include/libpman.h

```diff
@@ -66,6 +66,13 @@ extern "C"
 	 */
 	int pman_get_required_buffers(void);
 
+	/**
+	 * @brief Return whether modern bpf is supported by running kernel.
+	 *
+	 * @return supported true or false.
+	 */
+	bool pman_check_support();
```

Comment:
While i was at it, exported this function in the libpman header. Otherwise compilation complained about it.